includes/class-wc-payments-account.php

```diff
@@ -380,6 +380,25 @@ public function get_account_status_data(): array {
 		];
 	}
 
+	/**
+	 * Get structured account details including status, payout info, and banners for frontend display.
+	 *
+	 * @return null|array Account details array with nested status objects, or null if unavailable.
+	 */
+	public function get_account_details(): ?array {
+		$account = $this->get_cached_account_data();
+		if ( empty( $account['account_details'] ) ) {
+			return null;
+		}
+
+		$account_details = $account['account_details'];
+		if ( isset( $account_details['account_status'], $account_details['payout_status'], $account_details['banner'] ) ) {
```

Comment:
This happens to work because `banner` is null on my instance, but I think it should be inverse?
```suggestion
		if ( ! isset( $account_details['account_status'], $account_details['payout_status'], $account_details['banner'] ) ) {
```

Reply:
Good catch! Actually, I'd like to check and ensure the `banner` key exists but its value can be null too. Addressed here 2369318087812992e2d5b22d3c131457341dd82f